src/index.html

```diff
@@ -6,9 +6,90 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+      crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img
+          src="images/logo.png"
+          alt="moyo_logo"
+          class="logo_img"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a class="nav_link is-active"
+              href="#apple"
+            >
+              Apple
+            </a>
```

Comment:
please fix all similar cases
![image](https://user-images.githubusercontent.com/22824431/234234945-f0aec5ce-d360-4db5-95d9-c6d41061f557.png)

```suggestion
            <a class="nav_link is-active" href="#apple">
              Apple
            </a>
```